src/views/ReviewConfirm.vue

```diff
@@ -51,7 +51,7 @@
         <p class="mt-1">Enter the legal name of the person authorized to complete and submit this application.</p>
       </header>
       <div :class="{ 'invalid-section': isCertifyInvalid }">
-        <certify
+        <Certify
```

Comment:
FYI, changing this from kebab-case to TitleCase (ie, the component name) highlights it differently in my IDE (VS Code) and I can click on this to go to its definition - definitely a plus.